content/post/articles/music-while-coding-is-it-worth.md

```diff
@@ -0,0 +1,72 @@
+
+# For some, music is unuseful while coding
+
+I think liking the music or not liking it while coding depends on the song, task, and individual personality.
+There are several types of developers who use many methods to increase their productivity, their concentration we can retain 3 types of them:
+
+* *Those who need complete silence while they’re coding*
+* *Those who can work anywhere while just tuning out the noise in the background*
+* *Those who put on their headphones, throw on the perfect playlist, and get to work*
+
+The developer's main argument for saying music isn't helpful while coding is that it causes distraction, when you code you use your working memory to solve the problems you face, you use your mental bandwidth to manipulate multiple pieces of information at once.
+
+Several studies agree that it is not the music itself that is the problem but rather the lyrics, the volume, the speed, the type of the music.
+
+*(Sorry, I didn’t find studies that involved software developers, or computer science, let me know if you get something on the internet.)*
+
+* [Background music: Effects on attention performance](https://content.iospress.com/articles/work/wor01410)
+>*Abstract: Objective: Previous studies indicate that noise may affect worker attention. However, some background music in the work environment can increase worker satisfaction and productivity. This study compared how music with, and without, lyrics affects human attention. Participants: One hundred and two participants, aged 20–24 years, were recruited into this study. Fifty-six males and 46 females participated in this study. Methods: Background music with, and without lyrics, was tested for effects on listener concentration in attention testing using a randomized controlled trial (RCT) study. Results: The comparison results revealed that background music with lyrics had significant negative effects on concentration and attention. Conclusions: The findings suggest that, if background music is played in the work environment, music without lyrics is preferable because songs with lyrics are likely to reduce worker attention and performance.*
+
+* [Effects of background music on concentration of workers ](https://pubmed.ncbi.nlm.nih.gov/21508527/)
+>*Results: We conclude background music influenced listener attention. This influence has more to do with listener fondness for the music than with type of music. Compared to situations without background music, the likelihood of background music affecting test-taker attention performance is likely to increase with the degree to which the test-taker likes or dislikes the music.*
+
+
+Very well, dear reader (whether you are a man or a woman) thank you for reading until the end, see you soon in a future article.
+
```

Comment:
At the end, what is your own opinion? Can you please add your own thoughts on this subject?

Reply:
I am totally for the first opinion 'music is useful while coding', music really helps me to get in my code.
When I get a good vibe with music, everything is ok.

Reply:
Great. Can you put that as the last paragraph of your article? It's useful for readers to get an insight into the author's mind.

Reply:
Ok Fine! i can agree on that!